docs/installation/webservers/apache.md

```diff
@@ -0,0 +1,197 @@
+![SeAT](http://i.imgur.com/aPPOxSK.png)
+
+# webserver - apache
+
+The SeAT web interface requires a web server to serve the HTML goodies it has. Apache is an alternative of nginx. You don't **have** to use it, so if you prefer something else, feel free.
+
+This guide contains two sections:
+
+* Docker setup
+* Host Setup (f.e. via SeAT-Tool or manual setup)
+
+## docker setup
+
+This part is about how to setup apache to serve your seat-docker installation:
+
+!!! info "Docker-Setup"
+
+    This part is still under construction.
+    
+A minimal Apache config might look like this
+
+````
+<VirtualHost *:80>
+    ServerName discourse.example.com
+    ProxyPass / http://0.0.0.0:8080/
+    ProxyPassReverse / http://0.0.0.0:8080/
+</VirtualHost>
+````
+
+---
+
+## host installations
+In order to get the SeAT fronted running, we need to configure Apache to serve our SeAT installs `public/` folder. This is the only folder that should be internet facing. That small `index.php` is the gateway into the application.
+The Apache configuration itself will depend on how your server is set up. Generally, virtual hosting is the way to go, and this is what this guide will be showing here.
+
+
+### Unbuntu Installation
+
+
+Let's install Apache & PHP7.2 mod:
```

Comment:
If you are using PHP 7.1, just update the version number to reflect this for libapache2-mod-php7.1